main.tf

```
@@ -49,7 +49,7 @@ resource "google_cloud_run_service" "service" {
                 value = try(env.value.value, null)
 
                 dynamic "value_from" {
-                  for_each = try(env.value.values_from, [])
+                  for_each = try(env.value.value_from, [])
```

value from is an object.

```suggestion
                  for_each = try([env.value.value_from], [])
```